addons/mail/static/src/components/composer/composer.scss

```diff
@@ -184,19 +181,12 @@
     }
 }
 
-.o_Composer_button {
-    color: gray('600');
-
-    &.o-open{
-        background-color: gray('200');
-    }
-
-    &.o-composer-is-compact {
-        border-left: none;
-        border-radius: 0 3px 3px 0;
-    }
+.o_Composer_button.o-composer-is-compact {
+    border-left: none;
+    border-radius: 0 3px 3px 0;
 }
 
+
 // TODO FIXME o-open on the button should be enough, https://github.com/odoo/owl/issues/693
 .o_is_open {
     .o_Composer_button {
```

Comment:
Let's use this PR to also move this rulesets at the top of `// Style` category, and flatten selector:

```css
// -----------
// Style
// -----------

// TODO FIXME
.o_is_open .o_Composer_button {
   // ...
}
```

Also would clarify context of FIXME, like style of button when popover is "open" comes from `web.Popover`, and we can't define a modifier on `.o_Composer_button` due to not being aware of `Popover`'s state in context of template (+ keep link to OWL issue ofc).